docs/scala.md

```diff
@@ -52,44 +54,85 @@ e.g. ScalaTest, specs2, ScalaCheck, utest.
 
 * The [`--test_filter=<filter_expression>`](https://docs.bazel.build/versions/master/user-manual.html#flag--test_filter) option.
   * The syntax of the `<filter_expression>` varies by test framework, and not all test frameworks support the `test_filter` option at this time.
-  * For specs2, `<filter_expression>` simply matches full, `.`-separated classnames
+  * For specs2, `<filter_expression>` simply matches full `.`-separated classnames. Add the test name after the classname to run a single test.
     * example: `my.package.MyTest`
+    * example: `my.package.MyTest#remove trailing spaces from string`
```

Comment:
This kinda looks like a comment; could you wrap the last part with `< >` so people don't get confused?
i.e. `my.package.MyTest#<remove trailing spaces from string>`